docker/bionic/Dockerfile

```diff
@@ -20,6 +20,8 @@
 FROM ubuntu:bionic AS build_deps
 LABEL maintainer="Apache NiFi <dev@nifi.apache.org>"
 
+# MINIFI_OPTIONS will be passed directly to cmake
+# use it to define cmake options (e.g. -DENABLE_AWS=ON -DENABLE_AZURE=ON)
```

Comment:
I think this comment might be misplaced (also some other Dockerfiles)

Reply:
you are absolutely right :+1: 
fixed in https://github.com/apache/nifi-minifi-cpp/pull/1369/commits/6851250bc6ce26b4c7a0c5824e66cf8d7d16f923